src/main/java/com/dm/dmservicesandbox/domain/ServiceReponse.java

```diff
@@ -0,0 +1,15 @@
+package com.dm.dmservicesandbox.domain;
+
+import lombok.Value;
+
+@Value
+public class ServiceReponse {
+
+    private Boolean success;
```

Comment:
Boolean status perhaps better name

Reply:
I would rename to status if the value is either success or failure... if we keep a boolean, I think success is less confusing... maybe isSuccess

Reply:
I kept this as a placeholder.